WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -0,0 +1,580 @@
+
+    // Wrapper views
+    @IBOutlet private weak var innerContentView: UIView!
+    @IBOutlet private weak var cardContentView: UIView!
+    @IBOutlet private weak var cardBorderView: UIView!
+
+    // Header realated Views
+    @IBOutlet private weak var headerView: UIView!
+    @IBOutlet private weak var avatarImageView: UIImageView!
+    @IBOutlet private weak var blogNameButton: UIButton!
+    @IBOutlet private weak var bylineLabel: UILabel!
+    @IBOutlet private weak var menuButton: UIButton!
+
+    // Card views
+    @IBOutlet private weak var featuredMediaView: UIView!
+    @IBOutlet private weak var featuredImageView: UIImageView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var summaryLabel: UILabel!
+    @IBOutlet private weak var tagButton: UIButton!
+    @IBOutlet private weak var wordCountLabel: UILabel!
+    @IBOutlet private weak var attributionView: ReaderCardDiscoverAttributionView!
+
+    // Action buttons
+    @IBOutlet private weak var actionButtonRight: UIButton!
+    @IBOutlet private weak var actionButtonCenter: UIButton!
+    @IBOutlet private weak var actionButtonLeft: UIButton!
+    @IBOutlet private weak var actionButtonFlushLeft: UIButton!
+
+    // Layout Constraints
+    @IBOutlet private weak var featuredMediaHeightConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var featuredMediaBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var titleLabelBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var summaryLabelBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var attributionHeightConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var attributionBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var tagButtonHeightConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var tagButtonBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var wordCountBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var actionButtonViewHeightConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var actionButtonViewBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var cardContentBottomConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var maxIPadWidthConstraint: NSLayoutConstraint!
+
+    public weak var delegate: ReaderPostCellDelegate?
+    public weak var contentProvider: ReaderPostContentProvider?
+
+    private var featuredMediaHeightConstraintConstant: CGFloat = 0.0
+    private var featuredMediaBottomConstraintConstant: CGFloat = 0.0
+    private var titleLabelBottomConstraintConstant: CGFloat = 0.0
+    private var summaryLabelBottomConstraintConstant: CGFloat = 0.0
+    private var attributionBottomConstraintConstant: CGFloat = 0.0
+    private var tagButtonHeightConstraintConstant: CGFloat = 0.0
+    private var tagButtonBottomConstraintConstant: CGFloat = 0.0
+    private var wordCountBottomConstraintConstant: CGFloat = 0.0
```

Comment:
You can use `= CGFloat(0)` directly (so that it looks consistent the boolean assignments below)